services/query-builder.js

```diff
@@ -64,6 +64,8 @@ function QueryBuilder(model, opts, params) {
       if (params.sort.indexOf('.') !== -1) {
         // NOTICE: Sort on the belongsTo displayed field
         return [[opts.sequelize.col(params.sort), order]];
+      } else if (params.associationName) {
+        return [[opts.sequelize.col(`${params.associationName}.${params.sort}`), order]];
```

Comment:
This is a consequence of the implementation changes in `findQuery` function of the `HasManyGetter`.

This centralized code for `HasManyGetter` and `ResourcesGetter` will now behave a bit differently as the `ResourcesGetter` does not provide `params.associationName`. It will decrease the code maintainability.